addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2113,9 +2113,19 @@ var SnippetsMenu = Widget.extend({
             // TODO: adapt in master - used to hide XML 'img' options when image
             // is not supported.
             const xmlImageOption = !$style[0].hasAttribute('data-js') && (selector.indexOf('img') !== -1);
-            const nonSupportedImageSelector = '[data-oe-type="image"] > img';
+            const nonSupportedImageSelector = '[data-oe-type="image"] > img, [data-oe-xpath]';
```

Comment:
not related to your diff so just a suggestion / more just a question
Is `oe-type` supposed to be obvious that it is targetting a `t-field` instruction? It is not for me :o. Why wasn't this `[data-oe-field][data-oe-type="image"] > img` in the first place?
From what I got, images 

Same remark for the diff in `image_processing.js`.

Reply:
> what would you imagine to receive "image branding" if not a t-field instruction?

Well, it was the other way around: _"why is this targeting a t-field without targeting a t-field, am I missing something or is it targeting more than a t-field? t-field got the `oe-type`, but what else also got it if it didn't target the `oe-field`?"_

Reading
```js
img.parentElement && !img.parentElement.dataset.oeType
```

I was trying to figure why it wasn't
```js
img.parentElement && !img.parentElement.dataset.oeField
```
And tried to find what it was including on top of the `t-field`

Honestly, if `img.parentElement.dataset.oeType` is supposed to be obvious that it targets `t-field` instructions, I might be dumb :o

Maybe I am not playing around enough with the branding to automatically be sure that `oe-type` is bound to `oe-field` (it could have arguably be named `oe-field-type`), I figured that if I had to quickly check, others will have too :shrug: (and could be avoided with a comment or an extra attrib check).

Was just my 2 cents

Reply:
![image](https://user-images.githubusercontent.com/10338094/173546665-b777a668-fb39-443b-af0f-6bfdcceb48be.png)

data-oe-field does not mean it is a t-field :wink: Really the best indicator is data-oe-type, as data-oe-field is just present on any branding.

```javascript
img.parentElement && !img.parentElement.dataset.oeField
```
-> would prefer to edit images which are direct children of editable view sections :wink: 

Like I said, I guess a comment could be good (but would mean duplicating it everywhere), this will be refactored in master at some point.